README.rst

```diff
@@ -94,6 +94,15 @@ or
 
 $ ./host_os.py build-package --help
 
+* Build custom software
+
+The build code keeps a cache of all source code management (Git, Mercurial, SVN)
```

Comment:
"packages repositories" may be confused with yum repositories, as "repository" is a generic term used by VCS (git/mercurial/svn) and Linux packages management systems (yum/aptitude/zypper). I used "packages source code (Git/Mercurial/SVN) repositories", hope that has became more obvious to interpret.